kahuna/public/js/components/gr-icon/gr-icon.css

```diff
@@ -0,0 +1,27 @@
+.gr-icon {
```

Comment:
Could do, would mean we'd also have to have

```css
gr-icon i {
    font-style: normal;
}
```

To override the default browser styling of `<i>`.